### DISCUSSION TITLE
[Question] Is there any way to replace constant string in markdown file?

### DISCUSSION THREAD

**Discussion by DreamOfIce** [vuepress/core#1317] [Q&A] [answered]
Like https://cn.vitejs.dev/guide/env-and-mode.html#html-env-replacement, but this doesn't seem to work in markdown files though?

**Comment by DreamOfIce** [comment]
@Mister-Hope Is there any way to do this?

**Comment by Mister-Hope** [comment, marked as answer]
https://stackblitz.com/edit/vuepress-markdown-variable?file=package.json

I don' think template replacement could be aligned under 2 different bundler, so you should make them as variables.

**Comment by chrismcg14** [comment]
@Mister-Hope Would a preprocessor be required to process variables within markdown syntax? I.e 

 someString-$someVariable surrounded by ``

Is there a way to make that work using this approach or would we have to implement a preprocessor?
